services/.env.example

```diff
@@ -97,6 +99,14 @@ USERCONFIG_121_SERVICE_EMAIL_USER_KOBO=kobo-user@example.org
 USERCONFIG_121_SERVICE_PASSWORD_USER_KOBO=password
 
 
+# Redis connection details for queueing with Bull
+# For development, use the name of the docker-container (e.g. `121-redis`)
+# For production, use the connection to Azure Cache for Redis
+REDIS_HOST=
+# For local development, use 6379, for remote connections use 6380
+REDIS_PORT=
+# You can keep the password empty for local development
+REDIS_PASSWORD=
```

Comment:
There are all kinds of "sections" added to this file.
Why not add it to a relevant one?

This file is set up so that it can be read from top-to-bottom to configure everything correctly. Its not a "dump everything below the rest"-file.